tool/model/comparison.py

```diff
@@ -433,10 +459,22 @@ class ConsolidationReason:
 
 
 @dataclass(frozen=True)
-class ComparisonOutcome:
```

Comment:
Here's the (very minor) restructure of ComparisonOutcome into two seperate NodeComparisonOutcome + SpanComparisonOutcome

Everything else in this file is i think just auto-formatter doing it's thing